### DISCUSSION TITLE
Turborepo Bubble Up Help

### DISCUSSION THREAD

**Discussion by tgaeta** [vercel/turborepo#853] [Help] [answered]
Recently I was tasked to spinup a turborepo, called `turborepo-frontend`. It almost mirrors the default install folder structure. Now, a backend team recently merged in their code making the folder structure look like this:
```shell
monorepo/
├─ backend/
│  ├─ api/
├─ turborepo-frontend/
│  ├─ apps/
│  ├─ packages/
│  ├─ package.json
│  ├─ turbo.json
├─ infrastructure/
│  ├─ aws/
│  ├─ Pulumi.yaml
├─ .gitignore
├─ README.md
```
I honestly am not sure how to proceed. Do I need to add a `turbo.json` and `package.json` file to each new folder and at the root now?

**Comment by mehulkar** [comment, marked as answer]
This answer is pretty late, but if you're still looking for an answer... the easiest path is probably to move your `turbo.json` to the root of the repo. Alternatively, in Turbo 1.7, we introduced the ability to install `turbo` globally on your system. With that, you should be able to leave `turbo.json` where it is and `cd turborepo-frontend && turbo run build`. (I haven't tried this, but I think it should work).

**Reply by tgaeta** [reply]
No worries about the late reply. Yep, this is how we proceeded. It has grown significantly since Mar 2022.

**Reply by mehulkar** [reply]
Thanks for coming back here and replying! Just to be clear, which did you do? `cd...` method or moving turbo config to root?

**Reply by tgaeta** [reply]
turbo config to root
